helpdesk/teams/models.py

```diff
@@ -58,3 +58,32 @@ class Meta:
 
     def __str__(self) -> str:
         return f"Comment on {self.team.name} at {self.created_at} by {self.author}"
+
+
+class TeamAttendanceEventType(models.TextChoices):
+    ARRIVED = "AR", "Arrived"
+    LEFT = "LE", "Left"
+    DELAYED = "DE", "Delayed"
+    DROPPED_OUT = "DO", "Dropped Out"
+
```

Comment:
Nit: Since these are already small, might as well let the values be human readable

```suggestion
    ARRIVED = "ARRIVED", "Arrived"
    LEFT = "LEFT", "Left"
    DELAYED = "DELAYED", "Delayed"
    DROPPED_OUT = "DROPPED_OUT", "Dropped Out"

```